Add documentation for using appimage with Tails (#87)
spesmilo/electrum-docs: tails.rst

Using the most current Electrum on Tails
========================================

> Tails currently ships with an outdated and insecure version of Electrum that can no longer be used on the public Electrum server network as of March 2019. Unfortunately installed software on Tails cannot be permanently upgraded since it is a fixed (read-only) image - you have to wait for Tails to upgrade.

Reviewer: I would disagree with 3.1.3 being insecure, let's not say that.
```suggestion
Tails currently ships with an outdated version of Electrum that can no longer be used on the public Electrum server network as of March 2019. Unfortunately installed software on Tails cannot be permanently upgraded since it is a fixed (read-only) image - you have to wait for Tails to upgrade.
```

> However, you can use the most current Electrum with Tails that eliminates the phishing message and includes other improvements by using the appimage (a self-contained file that works on Tails linux). 

Reviewer: ```suggestion
However, you can use a recent version of Electrum with Tails by using the AppImage binary we distribute (a self-contained executable that works on x86_64 Linux (including Tails)). 
```

> These steps work on the current Tails (3.12.1) but may work on older versions too.

Reviewer: ```suggestion
These steps have been tested on Tails 3.12.1.
```

Author: Committed on my end
